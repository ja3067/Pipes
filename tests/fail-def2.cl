if False:
    def foo():
        return x

x = 3
print(x)
print(foo())
