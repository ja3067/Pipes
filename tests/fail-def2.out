3
RuntimeError: name 'foo' is not defined
